recipes/xpack/all/conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class XpackConan(ConanFile):
+    name = "xpack"
+    description = "C++ reflection, ability to convert between C++ structures and json/xml."
+    topics = ("xpack", "json", "reflection", "xml")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/xyz347/xpack"
+    license = "MIT"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="*", dst=os.path.join("include", "xpack"), src=self._source_subfolder)
```

Comment:
That's too much. There are folders and files you don't want in include dir, like gtest, example, .gitignore etc

What about thirdparty folder? Is it possible to unvendor rapidjson & rapidxml?

Reply:
I have exclude example and gtest dir. As for the thirdparty dir, I dont know if they are modified. Maybe I can exclude them and make them requirements.

Reply:
rapidjson and rapixml are included as "thirdparty/rapidjson/*" "thirdparty/rapidxml/*", so I cant get rid of them without making pr to xpack.